src/domdiv/draw.py

```diff
@@ -1993,12 +2008,16 @@ def drawText(self, item, panel, divider_text="card"):
 
         s = getSampleStyleSheet()["BodyText"]
         s.fontName = self.fontStyle["Rules"]
```

Comment:
I was surprised that this wasn't using the Minion font. Any objections to using `self.fontStyle["Regular"]` instead? It's a little bit of extra work to switch back to Times for the `<dash>` rendering, since in Minion the dashes have whitespace between them, and to register the font family so bold and italics work, but I've got code locally that does all of that.